### PR TITLE
Add Corn and Spicy Pepper

### PR DESCRIPTION
**Pull request by Mctalian**
Probably more of a placeholder and showing my interest in contributing ❤️ 

I don't have the webp files and it's possible some of the data is slightly incorrect due to unknowns, but this should be a good start!

### CODE REVIEW

File: assets/scripts/garden-planner/enums/cropcode.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ enum CropCode {
   Cotton = 'Co',
   Blueberry = 'B',
   Apple = 'A',
+  Corn = 'Cr',
+  SpicyPepper = 'SP',
```
Comment (VincentAmante) [inline assets/scripts/garden-planner/enums/cropcode.ts]: I should remember to leave a guide-line for adding new crops. These are good proposals, but the save-codes are separated by capital letters. Converting it to 'Sp' instead of just 'S' seems like a better idea though, so I'll add this in my own commit

File: stores/useHarvestOptions.ts
Comment (VincentAmante) [inline stores/useHarvestOptions.ts:1]: Oh thanks for seeing this, this wasn't actually connected to the code and was part of a process to save config information. This edit is amazing though and if you want to do another pull request then this would be nice to include.

**Comment by VincentAmante** [comment]
Heya! First of all thank you so much for making a pull request. I apologise that I haven't seen this till now, as I was also working on the same changes.

I unfortunately have to decline this one for a couple reasons, namely wanting to avoid incomplete additions merged to the main branch and also because I prefer to aggregate changes on either the hotfix or dev branch before merging it with Main. I ended up making commits with your crop-code suggestion and colour, but I'm also realising it's better if I had let you done that in another PR instead. 

So sorry about this, you're actually the first pull-request I've had so I'm making rookie mistakes. I'm going to add more documentation and guidelines for making changes outside of what's currently in CONTRIBUTING.md. In hindsight, there probably should be a dedicated branch for new patches.

**Review by VincentAmante — COMMENTED** [review]
Sadly might have to be declined as I'm already adding a highly overlapping commit. Highly appreciated though and sincere apologies for not seeing this much earlier

**Comment by VincentAmante** [comment]
Hello! Thinking back on it, as I believe I've fumbled this PR _and_ made a couple changes based off it, it's only right I credit you for code contribution on the planner website. I prefer your suggestion of the cropcode, as well as the colours for the UI.

Your PR was a good attempt at adding the new crops, it also exposed potential flaws in how I've documented my project which I will be promptly reconsidering

**Comment by Mctalian** [comment]
Sounds good! I figured being incomplete it would get closed so no worries or hard feelings! I'm glad you were able to draw some inspiration from it!

I noticed a few things that can probably be smoother with some repeated code, so I may look into that and put a PR together. Again, no expectations on accepting it, but I'm hoping to make things easier for you going forward 🤞🏼
